src/chai/ChaiMacros.hpp

```diff
@@ -45,7 +45,7 @@
 
 #include "chai/config.hpp"
 
-#if defined(CHAI_ENABLE_CUDA)
+#if defined(CHAI_ENABLE_CUDA) && defined(__CUDACC__)
```

Comment:
Nice. Have you run into this as a problem somewhere?

Reply:
Right now in geos. I tried to get around it by adding cuda as a dependency to everything but that led to other issues. We do something similar in axom:
https://github.com/LLNL/axom/blob/6d8f3cf2ff523678bf20a9df69c4dabc6b028c31/src/axom/core/Macros.hpp#L28

Reply:
Makes sense to me, thanks 👍